ember-cli-build.js

```diff
@@ -2,48 +2,11 @@
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-const environment = process.env.EMBER_ENV;
-const IS_PROD = environment === 'production';
-const IS_TEST = environment === 'test';
-
 module.exports = function (defaults) {
   let app = new EmberApp(defaults, {
-    autoImport: {
-      exclude: ['flandersMunicipalities.ts'],
-      webpack: {
-        node: {
-          global: true,
-        },
-      },
-      useSwcParser: true,
-      resolve: {
-        extensions: ['.js', '.json'],
-      },
-    },
-    hinting: IS_TEST,
-    tests: IS_TEST,
-    'ember-cli-babel': {
-      includePolyfill: IS_PROD,
-    },
-    autoprefixer: {
-      sourcemap: false,
-    },
```

Comment:
Not sure why this was needed. If we don't want the autoprefixing we should just remove the ember-cli-autoprefixer addon.